src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -406,38 +407,44 @@ export default defineComponent({
      */
     const isBoxed = computed(() => props.layout === 'box')
     const i18n = useI18n()
-    const layoutBasedProps = computed(() => {
-      if (!isBoxed.value) return {}
-      return {
-        href: `/audio/${props.audio.id}`,
-        class:
-          'block focus:bg-white focus:border-tx focus:ring-[3px] focus:ring-pink focus:ring-offset-[3px] focus:outline-none rounded-sm overflow-hidden cursor-pointer',
-      }
-    })
+    const layoutBasedProps = computed(() => ({
+      href: `/audio/${props.audio.id}`,
+      class:
```

Comment:
Should the `class` be a constant?

Reply:
True, moved out of `script` to the `class` attribute in the `template` in 03a2ae6.